README.md

```diff
@@ -26,7 +26,7 @@ There are 2 options to install this template:
 
 ### Deploy Using Oracle Resource Manager
 
-1. Click [![Deploy to Oracle Cloud](https://oci-resourcemanager-plugin.plugins.oci.oraclecloud.com/latest/deploy-to-oracle-cloud.svg)](https://cloud.oracle.com/resourcemanager/stacks/create?region=home&zipUrl=https://github.com/oracle-quickstart/oci-jboss-wildfly-atp/releases/latest/download/oci-jboss-latest.zip)
+1. Click [![Deploy to Oracle Cloud](https://oci-resourcemanager-plugin.plugins.oci.oraclecloud.com/latest/deploy-to-oracle-cloud.svg)](https://cloud.oracle.com/resourcemanager/stacks/create?region=home&zipUrl=https://github.com/oracle-quickstart/oci-jboss-wildfly-atp/releases/latest/download/oci-jboss-wildfly-atp-stack-latest.zip)
```

Comment:
The action creates the file `oci-jboss-latest.zip`
If you need to change the filename for your purpose, please rename it in the action.

Reply:
In OCI Architecture Center we are trying to name stacks with the name of the repo plus "stack-latest" suffix. We can make an exception for this rule here, just to protect your part of labs.

Reply:
this is not an issue with the lab: the file which name you changed here, is automatically created by a github action, so if you are changing the name here you also need to change it in the github action.

Reply:
Ok. I have returned to your original name.